docs/python-sdk/changelog.md

```diff
@@ -7,6 +7,50 @@ sidebar_position: 8
 
 # Changelog
 
+## v1.20.0 (2025-06-__)
+
+**[`fused-py`](/python-sdk/)**
+
+New Features:
+- Running a UDF with engine `local` will cache similarly to how it would when running on `remote`.
+- Large (batch) jobs can be accepted in a Pending state if they cannot start immediately.
```

Comment:
It's not clear to me what this "can be accepted" means. Accepted in which context?

Reply:
reworded it to try to make it clearer